### PR TITLE
Delete topics

### PR DESCRIPTION
**Pull request by andorp**
Fixes: #854

### CODE REVIEW

File: src/Frontend/Page/Topic.hs [hunk 1]
```diff
@@ -187,6 +189,14 @@ viewTopicHeaderDiv now ctx topic tab delegation = do
                                 a_ [id_ "edit-topic",  href_ $ U.editTopic space topicId] $ do
                                     i_ [class_ "icon-pencil"] nil
                                     "Thema bearbeiten"
+                        when canDeleteTopic .
+                            li_ [class_ "pop-menu-list-item m-form"] .
+                                div_ [class_ "pop-menu-list-item-form-wrapper"] $ do
+                                    i_ [class_ "icon-step-delete"] nil
```
Comment (andorp) [inline src/Frontend/Page/Topic.hs]: This icon is missing.

Comment (fisx) [inline src/Frontend/Page/Topic.hs]: can you open an issue, link to that line in the code, and assign to cc?

**Comment by fisx** [comment]
LGTM, with small nit-picks.  please take a look at my changes and merge if you and travis are green.

**Comment by andorp** [comment]
LGTM

**Pull request merged**